Tiltfile

```diff
@@ -0,0 +1,5 @@
+
+# If not using a standard local dev name, specify your k8s context here
+#allow_k8s_contexts('jenkins-dev')
+k8s_yaml(helm('charts/jenkins', values='charts/jenkins/values.yaml', set=['master.adminPassword=asdf']))
+watch_file('./charts/jenkins')
```

Comment:
Can we move this file to charts/jenkins? There have been some discussions if the helm chart for the jenkins-operator also moves to this repository and in that case it would be nicer to have it in the charts directory.

Reply:
Done!